cron/projects.txt

```diff
@@ -1,2314 +1,2314 @@
+github.com/PrismJS/prism,
+github.com/PrismLibrary/Prism,
+github.com/PyCQA/isort,
+github.com/PyCQA/pylint,
+github.com/PyGithub/PyGithub,
+github.com/PyO3/pyo3,
+github.com/PyTorchLightning/pytorch-lightning,
+github.com/Pylons/pyramid,
+github.com/Qiskit/qiskit-terra,
+github.com/QuantConnect/Lean,
+github.com/RIOT-OS/RIOT,
+github.com/RPCS3/rpcs3,
+github.com/RT-Thread/rt-thread,
+github.com/RTEMS/rtems,
+github.com/Radarr/Radarr,
+github.com/RailsEventStore/rails_event_store,
+github.com/RasaHQ/rasa,
+github.com/ReactTraining/react-router,
+github.com/ReactiveX/RxJava,
+github.com/ReactiveX/rxjs,
+github.com/RetroPie/RetroPie-Setup,
+github.com/RfidResearchGroup/proxmark3,
+github.com/RicoSuter/NJsonSchema,
+github.com/RicoSuter/NSwag,
+github.com/Rigellute/spotify-tui,
+github.com/RobotLocomotion/drake,
+github.com/RocketChat/Rocket.Chat,
+github.com/RubyMoney/money,
+github.com/Rust-SDL2/rust-sdl2,
+github.com/RustCrypto/traits,
+github.com/RustPython/RustPython,
+github.com/SAP/openui5,
+github.com/SCons/scons,
+github.com/SELinuxProject/selinux,
+github.com/STEllAR-GROUP/hpx,
+github.com/SWI-Prolog/swipl-devel,
+github.com/SchedMD/slurm,
+github.com/Security-Onion-Solutions/securityonion,
+github.com/Seldaek/monolog,
+github.com/SeleniumHQ/docker-selenium,
+github.com/SemanticMediaWiki/SemanticMediaWiki,
+github.com/SergioBenitez/Rocket,
+github.com/ServiceStack/ServiceStack,
+github.com/ShareX/ShareX,
+github.com/Shopify/sarama,
+github.com/SixLabors/ImageSharp,
+github.com/SkyAPM/cpp2sky,"project=envoy,depedency=true"
+github.com/Sonarr/Sonarr,
+github.com/SpecFlowOSS/SpecFlow,
+github.com/Spotifyd/spotifyd,
+github.com/Squidex/squidex,
+github.com/StackExchange/Dapper,
+github.com/StackExchange/StackExchange.Redis,
+github.com/StackStorm/st2,
+github.com/SteeltoeOSS/Steeltoe,
+github.com/Stellarium/stellarium,
+github.com/Submitty/Submitty,
+github.com/SubtitleEdit/subtitleedit,
+github.com/Supervisor/supervisor,
+github.com/Sylius/Sylius,
+github.com/System-IO-Abstractions/System.IO.Abstractions,
+github.com/TASVideos/BizHawk,
+github.com/TYPO3/TYPO3.CMS,
+github.com/TeXitoi/structopt,
+github.com/TechEmpower/FrameworkBenchmarks,
+github.com/Tencent/rapidjson,"project=envoy,dependency=true"
+github.com/TheAlgorithms/Python,
+github.com/TheBastionBot/Bastion,
+github.com/Thinstation/thinstation,
+github.com/Tribler/tribler,
+github.com/TrinityCore/TrinityCore,
+github.com/TryGhost/Ghost,
+github.com/TryGhost/Ghost-Admin,
+github.com/TykTechnologies/tyk,
+github.com/UPBGE/upbge,
+github.com/Ultimaker/Cura,
+github.com/Ultimaker/CuraEngine,
+github.com/Unidata/netcdf-c,
+github.com/UnigramDev/Unigram,
+github.com/Unity-Technologies/Graphics,
+github.com/Unity-Technologies/ml-agents,
+github.com/VSCodeVim/Vim,
+github.com/VSCodium/vscodium,
+github.com/Varying-Vagrant-Vagrants/VVV,
+github.com/VictoriaMetrics/VictoriaMetrics,
+github.com/VirtoCommerce/vc-platform,
+github.com/WAVM/WAVM,"project=envoy,dependency=true"
```

Comment:
Just a question since I'm unfamiliar with the metadata here. What do these mean exactly?

Reply:
The metadata is added to the results. https://github.com/ossf/scorecard/blob/6f2a0f43f48079ce5f2ad0db9cb2fd1e0c7aa221/e2e/executable_test.go#L35

Reply:
Thanks, I guess my question is more what `project=envoy,dependency=true` means exactly here. Does this mean envoy uses this as a dependency and relies on this for querying? 

Assuming that's what that means, this doesn't seem to scale, if every project needs to add themselves to their own dependencies here.

Reply:
It is anything that you can add to the results that the project would like to track. Here is an example where `envoy` would like to track its dependencies. 

The goal is to scorecard 10,000 + repo's also want to track critical repo's.

Reply:
Sure, this is fine for now but this usecase (envoy tracking dependencies this way) in particular doesn't seem particularly reliable or scalable so we'll likely need to change it.

Dependencies change frequently and we can't have every critical project manually add these labels here. It seems more reliable for envoy to compute its own list of dependencies and then checking scorecards from that list.

Reply:
https://github.com/ossf/scorecard/issues/289
https://github.com/ossf/scorecard/issues/287

Reply:
There tracking issues and discussions going with respect to this. This refactor is just so that we can get some data and clean up.

Reply:
Thanks for providing the context!